test/units/path_test.rb

```diff
@@ -73,6 +85,11 @@ def test_param_values_are_escaped
     assert_equal "https://demo.imgix.net/demo.png?hello_world=%2Ffoo%22%3E%20%3Cscript%3Ealert%28%22hacked%22%29%3C%2Fscript%3E%3C", ix_url
   end
 
+  def test_unicode_path_variants_are_utf8_encoded
```

Comment:
What about the tests for these paths?
```
"&$+,:;=?@#.jpg"

"ساندویچ.jpg"
```

Reply:
💯 , thanks for reminding me. Forgot to add these in.